src/webapi/Controllers/AccountController.cs

```diff
@@ -0,0 +1,217 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using eppeta.webapi.DTO;
+using eppeta.webapi.Identity.Data;
+using eppeta.webapi.Identity.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using OpenIddict.Abstractions;
+using OpenIddict.Validation.AspNetCore;
+
+namespace eppeta.webapi.Controllers;
+
+/*
+ * Original source:
+ * https://github.com/openiddict/openiddict-samples/blob/dev/samples/Hollastin/Hollastin.Server/Controllers/AccountController.cs
+ */
+
+[Authorize(AuthenticationSchemes = OpenIddictValidationAspNetCoreDefaults.AuthenticationScheme, Roles = Roles.Administrator)]
+[Route(Route)]
+public class AccountController : Controller
+{
+    public const string Route = "accounts";
+
+    private readonly UserManager<ApplicationUser> _userManager;
+    private readonly IIdentityRepository _identityRepository;
+    private readonly IOpenIddictTokenManager _tokenManager;
+
+    public AccountController(UserManager<ApplicationUser> userManager, IIdentityRepository identityRepository, IOpenIddictTokenManager tokenManager)
+    {
+        _userManager = userManager ?? throw new ArgumentNullException(nameof(userManager));
+        _identityRepository = identityRepository ?? throw new ArgumentNullException(nameof(identityRepository));
+        _tokenManager = tokenManager ?? throw new ArgumentNullException(nameof(tokenManager));
+    }
+
+    [HttpGet("{id}")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
```

Comment:
These three `ProducesResponseType` attributes help Swashbuckle know how to generate the Open API specification used by Swagger.